waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -5,26 +5,23 @@ module Wasp.Generator.DbGenerator.Jobs
   )
 where
 
-import Control.Concurrent (Chan, newChan, readChan, writeChan, writeList2Chan)
-import Control.Concurrent.Async (concurrently)
-import StrongPath (Abs, Dir, Path', (</>))
+import StrongPath (Abs, Dir, File', Path', Rel, (</>))
 import qualified StrongPath as SP
-import System.Exit (ExitCode (..))
 import qualified System.Info
-import Wasp.Generator.Common (ProjectRootDir, prismaVersion)
+import Wasp.Generator.Common (ProjectRootDir)
 import Wasp.Generator.DbGenerator.Common (dbSchemaFileInProjectRootDir)
-import Wasp.Generator.Job (JobMessage, JobMessageData (JobExit, JobOutput))
 import qualified Wasp.Generator.Job as J
 import Wasp.Generator.Job.Process (runNodeCommandAsJob)
 import Wasp.Generator.ServerGenerator.Common (serverRootDirInProjectRootDir)
 
--- `--no-install` is the magic that causes this command to fail if npx cannot find it locally
---    (either in node_modules, or globally in npx). We do not want to allow npx to ask
---    a user without prisma to install the latest version.
--- We also pin the version to what we need so it won't accidentally find a different version globally
---   somewhere on the PATH.
-npxPrismaCmd :: [String]
-npxPrismaCmd = ["npx", "--no-install", "prisma@" ++ show prismaVersion]
+-- | NOTE: The expectation is that `npm install` was already executed
+-- such that we can use the locally installed package.
+-- This assumption is ok since it happens during compilation now.
+prismaInServerNodeModules :: Path' (Rel ProjectRootDir) File'
+prismaInServerNodeModules = serverRootDirInProjectRootDir </> [SP.relfile|./node_modules/.bin/prisma|]
```

Comment:
Just to confirm -> so we don't use `npx` any more in order to avoid that bug, so instead we call the binary directly?
I guess it is tricky that this binary might move, right? with `npx prisma` we are sure we are going to get it, but here we are not I think? But I guess that is ok since we will catch it in tests anyway if it stops working?

Reply:
Good question. I think the two main risks to this working now but breaking in the future are:
1) prisma executable changes name
2) prisma executable changes location

For the first, I think the fact that we pin a specific prisma version should prevent this. For the second, I think this would be a pretty disruptive change by npm for the ecosystem.

npx I am guessing just assumes (or uses) the `npm bin` output when resolving locally from within a project (like `.wasp/out/server`):
```bash
$ npx which prisma
/home/shayne/dev/wasp/waspc/examples/todoApp/.wasp/out/server/node_modules/.bin/prisma

$ npm bin
/home/shayne/dev/wasp/waspc/examples/todoApp/.wasp/out/server/node_modules/.bin
```

the `npm bin` output is the same on WSL, Windows, and Mac in that it just appends `node_modules/.bin` to PWD.

But yes, the TLDR is our e2e test should detect this if this ever changes as well. Worst case, if this does not work as well as we want, we could add a script that has more logic in it to find corner cases as they arise. That may re-invent `npx` but at least we'd allow people to use the current latest with this change.